src/drawer.js

```diff
@@ -166,6 +166,38 @@ $.Drawer.prototype = {
         return this;
     },
 
+    /**
+     * This function gets the top left point from the viewport using the pixel point
```

Comment:
It's not really the top left point, is it? It's just whatever point you give it?

Reply:
Update documentation in a later commit.